website/docs/from_provider/quickstart.mdx

```diff
@@ -204,4 +209,6 @@ Following this guide, you *can* migrate towards codegen as a further step forwar
 [are composable by default]: /docs/from_provider/motivation#combining-providers-is-hard-and-error-prone
 [as mentioned above]: /docs/from_provider/quickstart#start-with-changenotifierprovider
 [Riverpod's `Consumer` APIs]: /docs/concepts/reading
-[lazy by default]: /docs/concepts/provider_lifecycles
\ No newline at end of file
+[lazy by default]: /docs/concepts/provider_lifecycles
+[paradigm shift]: /docs/from_provider/quickstart#migrate-one-provider-at-a-time
```

Comment:
This link feels weird. I don't see a relationship between "paradigm shift" and "migrate one provider at a time"

Reply:
Was this part of the PR, tho? 🤔
Anyways, I fixed this now.